src/main/java/uk/gov/hmcts/sptribs/ciccase/model/access/DefaultAccess.java

```diff
@@ -19,10 +19,10 @@ public class DefaultAccess implements HasAccessControl {
     public SetMultimap<HasRole, Permission> getGrants() {
         SetMultimap<HasRole, Permission> grants = HashMultimap.create();
         grants.putAll(CREATOR, Permissions.CREATE_READ_UPDATE);
-        grants.putAll(ST_CIC_CASEWORKER, Permissions.CREATE_READ_UPDATE);
-        grants.putAll(ST_CIC_SENIOR_CASEWORKER, Permissions.CREATE_READ_UPDATE);
-        grants.putAll(ST_CIC_HEARING_CENTRE_ADMIN, Permissions.CREATE_READ_UPDATE);
-        grants.putAll(ST_CIC_HEARING_CENTRE_TEAM_LEADER, Permissions.CREATE_READ_UPDATE);
+        grants.putAll(ST_CIC_CASEWORKER, Permissions.CREATE_READ_UPDATE_DELETE);
+        grants.putAll(ST_CIC_SENIOR_CASEWORKER, Permissions.CREATE_READ_UPDATE_DELETE);
+        grants.putAll(ST_CIC_HEARING_CENTRE_ADMIN, Permissions.CREATE_READ_UPDATE_DELETE);
+        grants.putAll(ST_CIC_HEARING_CENTRE_TEAM_LEADER, Permissions.CREATE_READ_UPDATE_DELETE);
```

Comment:
@sjamihmcts - Can we have the respective test file updated as well?

Reply:
Done